frontend/src/components/Login.vue

```diff
@@ -79,29 +79,15 @@ export default {
         password: this.password
       };
 
-      this.error_msg = '';
       if (!this.$v.$invalid) {
         this.showSpinner = true;
-        this.$store.dispatch('auth/login', loginData).then(
-          res => {
-            this.showSpinner = false;
-            this.clearForm();
-            if (res.status == 201) this.successAlert = true;
-          },
-          error => {
-            this.showSpinner = false;
-            this.clearForm();
-            const err = error.response.data.msg;
-
-            if (err == 'Not authorized') {
-              this.error_msg = 'Nieprawidłowa nazwa użytkownika, lub hasło';
-            } else if (err.includes('User already logged in')) {
-              this.error_msg = 'Jesteś już zalogowany';
-            } else if (error) {
-              this.error_msg = 'Logowanie nie powiodło się';
-            }
+        this.$store.dispatch('auth/login', loginData).then(status => {
+          this.showSpinner = false;
+          this.clearForm();
+          if (status == 201) {
+            this.successAlert = true;
```

Comment:
You should push $router to /dashboard here, because typing URL gets really annoying :D

Also while you at it, maybe you could figure out how to trigger submitting form on enter. I tried @keyup.enter="onSubmit" and it didn't work for some reason.

Reply:
Submitting form after enter press works, but redirect to Dashboard is surprisingly challenging, so I created new [ticket](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues/272) for that task

Reply:
Ok, so I close it